### PR TITLE
Added note about the apiVersion used up to 1.10.0

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
This change is [Reviewable](https://reviewable.io/reviews/jaegertracing/jaeger-operator/283)

**Comment by jpkrohling** [comment]
@objectiser, is this OK to be merged?

File: README.adoc [hunk 1]
```diff
@@ -20,13 +20,16 @@ To install the operator, run:
...
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing_v1_jaeger_crd.yaml # <2>
 kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/service_account.yaml
 kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role.yaml
 kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role_binding.yaml
 kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/operator.yaml
 ----
 <1> This creates the namespace used by default in the deployment files.
+<2> This installs the "Custom Resource Definition" for the `apiVersion: jaegertracing.io/v1`
+
+IMPORTANT: when using a Jaeger Operator up to v1.10.0, install the CRD file `io_v1alpha1_jaeger_crd.yaml` in addition to `jaegertracing_v1_jaeger_crd.yaml`. This is because up to that version, the `apiVersion` in use was `io.jaegertracing/v1alpha1`.
```
Comment (objectiser) [inline README.adoc]: Should this notice be included below aswell?

Comment (jpkrohling) [reply README.adoc]: Added. Our ultimate goal is to remove the OpenShift-specific instructions, but until that, we should indeed duplicate the instructions.

**Pull request merged**